tg_bot/modules/users.py

```diff
@@ -65,6 +69,133 @@ def broadcast(bot: Bot, update: Update):
                                             "due to being kicked.".format(failed))
 
 
+@run_async
+def restrict_group(bot: Bot, update: Update, args: List[str]) -> str:
+    message = update.effective_message  # type: Optional[Message]
+    chat_id_correct = False
+
+    # Check if there is only one argument
+    if not len(args) == 1:
+        message.reply_text("Incorrect number of arguments. Please use `/restrict chat_id`.",
+                           parse_mode=ParseMode.MARKDOWN)
+
+    chat_id = args[0]
+
+    # Check if chat_id is valid
+    if len(args) == 1 and not chat_id.startswith('-'):
+        message.reply_text("Invalid chat id! Make sure you include the '-' sign in the chat id.")
+
+    # Check if chat_id is in bot database
+    all_chats = sql.get_all_chats() or []
+    if len(args) == 1 and chat_id.startswith('-') and not any(chat.chat_id == chat_id for chat in all_chats):
+        message.reply_text("I can't seem to find the chat in my database. "
+                           "Use /chatlist to obtain a list of chats in my database.")
+
+    if len(args) == 1 and chat_id.startswith('-') and any(chat.chat_id == chat_id for chat in all_chats):
```

Comment:
All your chaining of if statements is useless, youre just doing extra checks for nothing